gallery/src/components/NxDrawer/NxDrawerWithNxFormExample.tsx

```diff
@@ -53,7 +53,7 @@ export default function NxDrawerWithNxFormExample() {
               <NxStatefulTextInput />
             </NxFormGroup>
             <NxFormGroup label="Country" sublabel="Pick your favorite from the list">
-              <NxFormSelect value={country} onChange={evt => setCountry(evt.target.value)}>
+              <NxFormSelect value={country} onChange={val => setCountry(val)}>
```

Comment:
```suggestion
              <NxFormSelect value={country} onChange={setCountry}>
```

Reply:
> <NxFormSelect value={country} onChange={setCountry}>

Batched this fix with the other onChange updates: [d11bc6c](https://github.com/sonatype/sonatype-react-shared-components/pull/1161/commits/d11bc6c768c2e0f8389587f945fa7b72f04fb553)